README.md

```diff
@@ -47,21 +47,25 @@ python -m api
 
 
 ### Run Bot
-[Bot README.md](src/bot/README.md) \
-1. Install the necessary python packages with
+[Bot README.md](src/bot/README.md)
+1. Navigate into the `src/bot` directory with
 ```cmd
-pip install -r src/api/requirements.txt
+cd src/bot
 ```
-2. Add the necessary environment variables in your local `.env`
+2. Install the necessary python packages with
+```cmd
+pip install -r requirements.txt
+```
+3. Add the necessary environment variables in your local `.env`
 These can be found in `src/bot/.env.example` and are listed below as well:
 
 ```
 TOKEN=
 API_URL=
 ```
-3. Run the bot with
+4. Run the bot with
 ```cmd
-python -m bot
+python __main__.py
 ```
```

Comment:
I would expect the bot to be run as a module and the documentation should definitely show this.

```cmd
Python -m bot
```

While in src/

Reply:
while being in the directory and using `__main__.py` should basically be the same,
I would prefer the cleaner way and the intended way to be shown in the README

Reply:
Sure, I'll revert those changes.

Reply:
I've reverted the changes, please go through the changes whenever possible.